apps/frontend/src/app/backups-overview-page/component/data-stores-panel/data-stores.component.ts

```diff
@@ -61,4 +61,21 @@ export class DataStoresComponent implements OnDestroy, OnInit {
       (a, b) => this.getFilledPercentage(b) - this.getFilledPercentage(a)
     );
   }
+
+  getOverflowTimeLabel(overflowTime: number | undefined): string {
+    if (overflowTime === undefined) {
+      return 'N/A';
+    }
+
+    if (overflowTime === 1) {
+      return `${overflowTime} day`;
+    } else if (overflowTime > 365) {
+      // const years = (overflowTime / 365).toFixed(1);
+      // return `${years} years`;
+      return `> 1 year`;
+    } else if (overflowTime == 0) {
```

Comment:
```suggestion
    } else if (overflowTime === 0) {
```
You should always use type safe comparison in TypeScript